src/abieos_numeric.hpp

```diff
@@ -159,9 +159,9 @@ template <typename T, auto size, typename std::enable_if<sizeof(T) == 1>::type*
 std::string binary_to_base58(const std::array<T, size>& bin) {
     std::string result("");
     for (auto byte : bin) {
-        int carry = byte;
+        unsigned int carry = byte;
```

Comment:
Still doesn't solve it. See https://stackoverflow.com/questions/21769068/casting-negative-integer-to-larger-unsigned-integer/21769421